src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -557,13 +558,9 @@ bool emberAfThermostatClusterSetpointRaiseLowerCallback(app::CommandHandler * co
     int8_t DeadBand                          = 0;
     uint32_t FeatureMap                      = 0;
 
-    // TODO re-enable reading reaturemap once https://github.com/project-chip/connectedhomeip/pull/9725 is merged
-#ifndef FEATURE_MAP_OVERIDE
-    emberAfReadServerAttribute(endpoint, Thermostat::Id, chip::app::Clusters::Globals::Attributes::Ids::FeatureMap,
-                               (uint8_t *) &FeatureMap, sizeof(FeatureMap));
-#else
-    FeatureMap = FEATURE_MAP_OVERIDE;
-#endif
+    emberAfReadServerAttribute(aEndpointId, chip::app::Clusters::Thermostat::Id,
```

Comment:
@bzbarsky-apple - I would like to use accessors here but do not know how. Help  would be appreciated.

Reply:
Need to fix accessors. https://github.com/project-chip/connectedhomeip/pull/12808

Reply:
@bzbarsky-apple @andy31415 Thanks.  I will  rebase this branch, and test this change shortly, then promote to non draft.